formlibrary/views.py

```diff
@@ -476,6 +481,29 @@ def get(self, request, *args, **kwargs):
         return JsonResponse(final_dict, safe=False)
 
 
+class TrainingParticipantListObjects(ListAPIView):
+
+    serializer_class = BeneficiarySerializer
+    pagination_class = CustomPagination
+
+    def get_queryset(self):
+        pk = int(self.kwargs['pk'])
+        return Beneficiary.objects.filter(
+            training__pk=pk
+        ).prefetch_related('training')
```

Comment:
Let's remove this function and try using `lookup_field = 'training__pk'` if it works.

Reply:
```
class TrainingParticipantListObjects(ListAPIView):

    serializer_class = BeneficiarySerializer
    pagination_class = CustomPagination
    lookup_field = 'training__pk'
    queryset = Beneficiary.objects.prefetch_related('training')
```
I've tried this but it doesn't filter anymore.

Reply:
leave out the queryset part

Reply:
I get the following error `TrainingParticipantListObjects' should either include a `queryset` attribute, or override the `get_queryset()` method`